driver-embedded/src/main/com/mongodb/embedded/client/MongoEmbeddedSettings.java

```diff
@@ -57,13 +100,19 @@ public static Builder builder(final MongoEmbeddedSettings settings) {
     @NotThreadSafe
     public static final class Builder {
         private String libraryPath;
+        private String yamlConfig;
+        private LogLevel logLevel = LogLevel.STDOUT;
```

Comment:
Although `STDOUT` is the server default, I thought that `LOGGER` is a better default for the library.